### PR TITLE
PWGCF: Recevice general GFW input from configurable

### PR DESCRIPTION
**Pull request by Luzhiyongg**

### CODE REVIEW

File: PWGCF/Flow/Tasks/FlowTask.cxx [hunk 1]
```diff
@@ -302,6 +318,16 @@ struct FlowTask {
     corrconfigs.push_back(fGFW->GetCorrelatorConfig("refN10 {3 2} refP10 {-3 -2}", "Ch10Gap3232", kFALSE));
     corrconfigs.push_back(fGFW->GetCorrelatorConfig("refN10 {4 2} refP10 {-4 -2}", "Ch10Gap4242", kFALSE));
     corrconfigs.push_back(fGFW->GetCorrelatorConfig("refN10 {2 2} refP10 {-2 -2}", "Ch10Gap24", kFALSE));
+    if (!UserDefineGFWCorr.empty() && !UserDefineGFWName.empty()) {
+      LOGF(info, "User adding GFW CorrelatorConfig:");
+      // attentaion: here we follow the index of cfgUserDefineGFWCorr
+      for (int i = 0; i < UserDefineGFWCorr.size(); i++) {
+        if (i >= UserDefineGFWName.size())
+          continue;
```
Comment (victor-gonzalez) [inline PWGCF/Flow/Tasks/FlowTask.cxx:326]: Shouldn't this condition give a `fatal` to the user telling him that something has been wrongly configured?

File: PWGCF/Flow/Tasks/FlowTask.cxx [hunk 2]
```diff
@@ -613,6 +639,9 @@ struct FlowTask {
       // magnet field dependence cut
       Magnetfield = getMagneticField(bc.timestamp());
     }
+    float independent = cent;
+    if (cfgUseNch)
+      independent = static_cast<float>(tracks.size());
```
Comment (victor-gonzalez) [inline PWGCF/Flow/Tasks/FlowTask.cxx:644]: Do you expect only 90 Nch as maximum? 
That are the limits of your histograms

**Review by victor-gonzalez — DISMISSED** [review]
Please, have a look at my comments in case you want to consider them

File: PWGCF/Flow/Tasks/FlowTask.cxx (hunk 1 above)
Comment (Luzhiyongg) [reply PWGCF/Flow/Tasks/FlowTask.cxx:326]: sure, I will add fatal signal here

File: PWGCF/Flow/Tasks/FlowTask.cxx (hunk 2 above)
Comment (Luzhiyongg) [reply PWGCF/Flow/Tasks/FlowTask.cxx:644]: No. 
The user should change the `axisIndependent` if he/she turn the` cfgUseNch` on
` ConfigurableAxis axisIndependent{"axisIndependent", {VARIABLE_WIDTH, 0, 5, 10, 20, 30, 40, 50, 60, 70, 80, 90}, "centrality axis for histograms"};`

**Comment by Luzhiyongg** [comment]
Hi victor, I push a new commit so the auto-merge was disabled.
Could you approve this PR again?

Best Regards,
Zhiyong

**Comment by Luzhiyongg** [comment]
Hi victor @victor-gonzalez , I push a new commit so the auto-merge was disabled.
Could you approve this PR again?

Best Regards,
Zhiyong

**Review by victor-gonzalez — APPROVED** [review]

**Pull request merged**